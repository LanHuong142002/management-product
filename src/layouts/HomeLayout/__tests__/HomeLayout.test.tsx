import { render } from '@testing-library/react';
import { HomeLayout } from '@layouts';

describe('Testing DetailsLayout', () => {
  it('Should renders DetailsTitle and DetailsBody correctly', () => {
    const header = <h1>Home Header</h1>;
    const body = <p>Home Body</p>;

    const { getByText, container } = render(<HomeLayout header={header}>{body}</HomeLayout>);

    expect(getByText('Home Header')).toBeInTheDocument();
    expect(getByText('Home Body')).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
