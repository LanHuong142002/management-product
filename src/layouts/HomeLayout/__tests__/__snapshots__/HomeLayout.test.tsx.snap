// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing DetailsLayout Should renders DetailsTitle and DetailsBody correctly 1`] = `
<div>
  <main
    class="home-wrapper"
  >
    <div
      class="home-header"
    >
      <h1>
        Home Header
      </h1>
    </div>
    <div
      class="home-body"
    >
      <p>
        Home Body
      </p>
    </div>
  </main>
</div>
`;
